pkg/conduit/plugin.go

```diff
@@ -22,6 +22,7 @@ const (
 	S3Plugin        = "builtin:s3"
 	PostgresPlugin  = "builtin:postgres"
 	LogPlugin       = "builtin:log"
+	FilePlugin      = "builtin:file"
```

Comment:
Is this meant to be a permanent change? I recall that we made a decision to remove this since it might be abused.

Reply:
@hariso nah I removed it.